src/components/not-found-results/NotFoundResults.tsx

```diff
@@ -0,0 +1,38 @@
+import { FC } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import Box from '@mui/material/Box'
+
+import AppButton from '~/components/app-button/AppButton'
+import { styles } from '~/components/not-found-results/NotFoundResults.styles'
+import howItWorksStudentSecond from '~/assets/img/find-offer/search_icon.svg'
+import { VariantEnum } from '~/types'
+import ImgTitleDescription from '~/components/img-title-description/ImgTitleDescription'
+
+interface NotFoundResultsProps {
+  description: string
+  buttonName: string
+}
+
+const NotFoundResults: FC<NotFoundResultsProps> = ({
+  description,
```

Comment:
provide onClick for the button, do it not required so you can omit passing it in your component

Reply:
Fixed